Firmware/MotorControl/controller.hpp

```diff
@@ -21,15 +21,16 @@ class Controller : public ODriveIntf::ControllerIntf {
     struct Config_t {
         ControlMode control_mode = CONTROL_MODE_POSITION_CONTROL;  //see: ControlMode_t
         InputMode input_mode = INPUT_MODE_PASSTHROUGH;  //see: InputMode_t
-        float pos_gain = 20.0f;                         // [(counts/s) / counts]
-        float vel_gain = 0.2f / 10000.0f;               // [Nm/(counts/s)]
+        float pos_gain = 20.0f;                         // [(rad/s) / rad]
+        float vel_gain = 0.2f / 7.7f;                   // [Nm/(rad/s)]
         // float vel_gain = 0.2f / 200.0f,              // [Nm/(rad/s)] <sensorless example>
-        float vel_integrator_gain = 0.4f / 10000.0f;    // [Nm/(counts/s * s)]
-        float vel_limit = 20000.0f;                     // [counts/s] Infinity to disable.
+        float vel_integrator_gain = 0.4f / 7.7f;        // [Nm/(rad/s * s)]
+        float vel_limit = 4.0f * M_PI;                  // [rad/s] Infinity to disable.
         float vel_limit_tolerance = 1.2f;               // ratio to vel_lim. Infinity to disable.
-        float vel_ramp_rate = 10000.0f;                 // [(counts/s) / s]
+        float vel_ramp_rate = 2.0f * M_PI;              // [(rad/s) / s]
         float torque_ramp_rate = 0.01f;                 // Nm / sec
         bool setpoints_in_cpr = false;
+        float circular_setpoint_range = 2.0f * M_PI;    //circular space if setpoints_in_cpr is used for controller
```

Comment:
Our convention is to have a space between the `//` and the comment. Usually also start with capital letter.

Reply:
Fixed in a262688